src/transaction/log_manager.c

```diff
@@ -3749,15 +3751,17 @@ log_append_schema_modification_lock (THREAD_ENTRY * thread_p, const OID * classo
     }
 
   LOG_PRIOR_NODE *node =
-    prior_lsa_alloc_and_copy_data (thread_p, LOG_SCHEMA_MODIFICATION_LOCK, RV_NOT_DEFINED, NULL, 0, NULL, 0, NULL);
+    prior_lsa_alloc_and_copy_data (thread_p, LOG_LOCKED_OBJECT, RV_NOT_DEFINED, NULL, 0, NULL, 0, NULL);
   if (node == nullptr)
     {
       assert (false);
       return;
     }
 
-  auto record = (LOG_REC_SCHEMA_MODIFICATION_LOCK *) node->data_header;
+  auto record = (LOG_REC_LOCKED_OBJECT *) node->data_header;
```

Comment:
Nit:
```suggestion
  LOG_REC_LOCKED_OBJECT *const record = (LOG_REC_LOCKED_OBJECT *) node->data_header;
```
Not sure if `auto *const record` is an option. I forgot these details.